### PR TITLE
Support Python 3.13

### PR DESCRIPTION
**Pull request by edgarrmondragon**
Wheel build passing in https://github.com/edgarrmondragon/py-setproctitle/actions/runs/10895982820.

### CODE REVIEW

**Comment by edgarrmondragon** [comment]
@dvarrazzo let me know if you'd like to see something else in this PR or if it's good to merge 😄

**Comment by dvarrazzo** [comment]
Hello, thank you for this!

I am sorry, this MR skipped under the radar. I don't have a lot of bandwidth right now, but I will try to take a look at it!

**Comment by edgarrmondragon** [comment]
ping @dvarrazzo 🙏

**Comment by dvarrazzo** [comment]
Trying a build. If it passes I'll release the package.

Thank you very much!

File: .github/workflows/packages.yml [hunk 1]
```diff
@@ -115,31 +131,33 @@ jobs:
...
 
       - name: Set up QEMU for multi-arch build
-        uses: docker/setup-qemu-action@v2
+        uses: docker/setup-qemu-action@v3
 
       - name: Build wheels
-        uses: pypa/cibuildwheel@v2.16.1
+        uses: pypa/cibuildwheel@v2.21.2
         env:
           CIBW_BUILD: pp*
```
Comment (edgarrmondragon) [inline .github/workflows/packages.yml:155]:
```suggestion
          CIBW_BUILD: pp38 pp39 pp310
```

File: .github/workflows/packages.yml [hunk 2]
```diff
@@ -92,14 +108,14 @@ jobs:
...
     steps:
       - name: Checkout repos
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
 
       - name: Build wheels
-        uses: pypa/cibuildwheel@v2.16.1
+        uses: pypa/cibuildwheel@v2.21.2
         env:
           CIBW_BUILD: pp*
```
Comment (edgarrmondragon) [inline .github/workflows/packages.yml:120]:
```suggestion
          CIBW_BUILD: pp38 pp39 pp310
```

**Review by edgarrmondragon — COMMENTED** [review]
It seems cibw is still trying to build `pp37` wheels

**Comment by edgarrmondragon** [comment]
Wheel builds are passing on my fork: https://github.com/edgarrmondragon/py-setproctitle/actions/runs/11860299695/job/33064232120.

But there's a failure in the `macos-native-python-test` job: https://github.com/dvarrazzo/py-setproctitle/actions/runs/11860296122/job/33055225047?pr=139. I might not have time to look at it today.

**Comment by dvarrazzo** [comment]
Everything seems passing for me. I had already fixed the leftover 3.7 builds in my branch.

I have also added a merge step to download together all the artifacts and bumped package version numbers. Force-pushing here in order to merge the MR leaving the attribution of your changesets.

**Comment by edgarrmondragon** [comment]
Nice, thanks @dvarrazzo!

**Comment by dvarrazzo** [comment]
setproctitle 1.3.4 released, with Python 3.13 support added.

**Pull request merged**